--- uade-2.03/configure.orig	2006-08-27 10:56:14.000000000 +0100
+++ uade-2.03/configure	2006-09-07 02:34:25.000000000 +0100
@@ -309,8 +309,8 @@
     if test -z "$prefix"; then
 	prefix="/usr/local"
     fi
-    uadedatadir="$prefix/share/uade2"
-    uadelibdir="$prefix/lib/uade2"
+    uadedatadir="$prefix/share/uade"
+    uadelibdir="$libdir/uade"
     if test -z "$bindir"; then
 	bindir="$prefix/bin"
     fi
@@ -329,7 +329,7 @@
 else
     # user installation
     if test -z "$prefix"; then
-	prefix="$HOME/.uade2"
+	prefix="$HOME/.uade"
     fi
     uadedatadir="$prefix"
     uadelibdir="$prefix"
@@ -391,7 +391,7 @@
     AOLIBS="`$PKG_CONFIG --libs ao`"
 fi
 
-pkgconfigdir="$prefix/lib/pkgconfig"
+pkgconfigdir="$libdir/pkgconfig"
 rm -f uade.pc
 if test "$PKG_CONFIG" != "" ; then
     installuadepcrule=""
